components/emoji/components/delete_emoji_modal.jsx

```diff
@@ -8,6 +8,11 @@ import {FormattedMessage} from 'react-intl';
 import DeleteModalTrigger from '../../delete_modal_trigger.jsx';
```

Comment:
Please change to ``import DeleteModalTrigger from 'components/delete_modal_trigger.jsx';``

Reply:
@saturninoabril , Actually, DeleteModalTrigger test casses cover the tests for the childClasses extending it.

Reply:
@saturninoabril , i will take care of the the other changes mentioned by you. Thanks